presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -1340,7 +1340,8 @@ Time Travel
 Iceberg and Presto Iceberg connector support time travel via table snapshots
 identified by unique snapshot IDs. The snapshot IDs are stored in the ``$snapshots``
 metadata table. You can rollback the state of a table to a previous snapshot ID.
-It also supports time travel query using VERSION (SYSTEM_VERSION) and TIMESTAMP (SYSTEM_TIME) options.
+It also supports time travel query using VERSION (SYSTEM_VERSION), BRANCH (SYSTEM_VERSION), TAG (SYSTEM_VERSION)
+and TIMESTAMP (SYSTEM_TIME) options.
```

Comment:
```suggestion
It also supports time travel query using SYSTEM_VERSION (VERSION) and SYSTEM_TIME (TIMESTAMP) options.
```

I would leave the branch and tag syntax to the section below.  But I would change the wording slightly to emphasize that our preferred syntax is SYSTEM_VERSION and SYSTEM_TIME.